dell_power_collector/README.md

```diff
@@ -2,43 +2,43 @@
 
 This agent collects metric and log data from Dell Power products and sends it to Insightfinder.
 
-Current supported product: powerflex, powerflex manager, powerstore, powerscale. For metric key, no nested structure is supported. For one specific key, if it has a nested structure, all its children will be collected. The metric collection will only be effective for number type value.
+Current supported products: powerflex, powerflex manager, powerstore, and powerscale. For metric keys, nested structure is not currently supported. For one specific key, if it has a nested structure, all of its children will be collected. The metric collection will only be effective for numerical-typed values.
 
 ## Build the Agent
 
-The agent is written in Go. To build the agent, you need to install Go 1.13 or later. The build from these 2 commends should be run in Linux environment. We don't support runnning in Windows environment yet. 
+The agent is written in Go. To build the agent, you need to install Go version 1.13 or later. The build from the following 2 commands should be run in a Linux environment. NOTE: We currently do not support running the application in Windows environments. 
 
 Run the following commands to build:
 ```bash
 # build on Linux (Recommend)
 ./build_for_linux_in_linux_env.sh
 
-# build on Windows
+# build on Windows (Not currently supported)
```

Comment:
The case is that we can build in windows environment but the application built from it can only be run in Linux env. This command is used to build in windows env for linux. We might want to update the NOTE above to be more clear on this.